claims.go

```diff
@@ -240,11 +256,11 @@ func verifyAud(aud []string, cmp string, required bool) bool {
 	return result
 }
 
-func verifyExp(exp *time.Time, now time.Time, required bool) bool {
+func verifyExp(exp *time.Time, now time.Time, required bool, clockSkew time.Duration) bool {
 	if exp == nil {
 		return !required
 	}
-	return now.Before(*exp)
+	return now.Before((*exp).Add(+clockSkew))
```

Comment:
Wouldn't it be more logically to Add/Sub the clock skew from `now`? Same goes for verifyNbf

Reply:
The math is sort of backward, decided to leave it alone. Unit tests confirm this is accurate.

Reply:
Should we call it leeway as a parameter then? Similar to the validator option. Then it would also make sense to add it to exp? What do you think?

Reply:
Yes, we could I used skew here because that is typically how it is described in the literature and I felt it would make it clear in the code and retained WithLeeway because that is how the original library had it.